parsec/core/gui/custom_dialogs.py

```diff
@@ -191,20 +191,42 @@ def get_text_input(
 
 
 class QuestionWidget(QWidget, Ui_QuestionWidget):
-    def __init__(self, message, button_texts, radio_mode=False):
+    def __init__(self, message, button_texts, radio_mode=False, oriented_question=False):
         super().__init__()
         self.setupUi(self)
         self.status = None
         self.dialog = None
         self.label_message.setText(message)
-        for text in button_texts:
-            b = Button(text)
+
+        if oriented_question:
+            no_text, yes_text = button_texts
+            assert not radio_mode
+
+            # Add "yes" button
+            b = Button(yes_text)
             b.clicked_self.connect(self._on_button_clicked)
             b.setCursor(Qt.PointingHandCursor)
-            if radio_mode:
-                self.layout_radios.addWidget(b)
-            else:
-                self.layout_buttons.insertWidget(1, b)
+            b.setStyleSheet(
+                "QPushButton {background-color: darkgrey;} QPushButton:hover {background-color: grey;}"
```

Comment:
A bit worried that it makes it look like a disabled button. It should be ok in this context since I don't think there's a case where a button in the question dialog can be disabled.

Reply:
I think its clear enough given the button change when it is hovered (which wouldn't be the case for a disabled button)